UDLs/6502Assembly_byCarlMyerholtz.xml

```diff
@@ -0,0 +1,67 @@
+<NotepadPlus>
+    <UserLang name="6502 Assembly" ext="s cba" udlVersion="2.1">
+        <Author>
+            <field name="Carl Myerholtz W9OMS"

```

Comment:
This tag is never closed.  Please fix it

Reply:
This is not a validate xml file. I wonder if it's tested.